homeassistant/components/yale_smart_alarm/coordinator.py

```diff
@@ -40,14 +45,28 @@ async def _async_update_data(self) -> dict:
             if device["type"] == "device_type.door_lock":
                 lock_status_str = device["minigw_lock_status"]
                 lock_status = int(str(lock_status_str or 0), 16)
+                jammed = (lock_status & 48) == 48
                 closed = (lock_status & 16) == 16
                 locked = (lock_status & 1) == 1
                 if not lock_status and "device_status.lock" in state:
                     device["_state"] = "locked"
+                    device["_state2"] = "unknown"
                     locks.append(device)
                     continue
                 if not lock_status and "device_status.unlock" in state:
                     device["_state"] = "unlocked"
+                    device["_state2"] = "unknown"
```

Comment:
Where is this information used?

Reply:
It isn't at the moment. The PR for doorlock will be added as soon as this PR is merged and it will use this information